quickstart/csharp/uwp/virtual-assistant/README.md

```diff
@@ -1,18 +1,12 @@
-# Quickstart: Create a voice-first virtual assistant with the Speech SDK in C# UWP
+# Quickstart: Create a Voice Assistant with the Speech SDK in C# UWP
 
-This quickstart will connect to a previously authored and configured bot to enable a voice-first virtual assistant experience from the client application.
-See the [accompanying article](https://docs.microsoft.com/azure/cognitive-services/speech-service/quickstart-virtual-assistant-csharp-uwp) on the SDK documentation page which describes how to build this sample from scratch in Visual Studio.
+This quickstart will connect to a previously authored and configured bot to enable a voice assistant experience from the client application.
+See the [accompanying article](https://docs.microsoft.com/en-us/azure/cognitive-services/speech-service/quickstart-voice-assistant-csharp-uwp) on the SDK documentation page which describes how to build this sample from scratch in Visual Studio.
```

Comment:
>en-us [](start = 58, length = 5)

Same comment here about language choice hard coded in url.
I'm not going to lag the others place, but please take a look at all of them.

Thanks,